slices/tzdata-legacy.yaml

```diff
@@ -0,0 +1,271 @@
+package: tzdata-legacy
+
+essential:
+  - tzdata-legacy_copyright
+
+slices:
+  # The "base" slice contains Canonical timezone abbreviations. These are
+  # primary, preferred zone names that are often used as abbreviations for
+  # location-specific timezones across the globe. Example: Europe/Sofia observes
+  # EET.
+  base:
+    essential:
+      - tzdata_base
+    contents:
+      /usr/share/zoneinfo/right/CET:
+      /usr/share/zoneinfo/right/CST6CDT:
+      /usr/share/zoneinfo/right/EET:
+      /usr/share/zoneinfo/right/EST:
+      /usr/share/zoneinfo/right/EST5EDT:
+      /usr/share/zoneinfo/right/Factory:
```

Comment:
Should we move this to etc? It's an odd timezone.

Reply:
Let's keep this here, because `etc` consists of files in `Etc/` only.

Reply:
Also because of symmetry, the same structure is maintained in other releases.